main.go

```diff
@@ -457,7 +462,7 @@ func transformCompile(args []string) ([]string, error) {
 		return nil, err
 	}
 	deferred = append(deferred, func() error {
-		return os.RemoveAll(tempDir)
+		return nil //return os.RemoveAll(tempDir)
```

Comment:
You probably used that for debugging next time maybe use  `-debugdir=debug`, please revert.

Reply:
Ok